### ISSUE TITLE
Feature Request: Add definition of 'trie' in create dictionary script.

### ISSUE THREAD

**Issue by KieranRobson** [streetsidesoftware/cspell-dicts#2386]
When using `pnpm run create-dictionary`, an option comes up as `Store as Trie`. 

I think it woulod be good to add a description to this, to suggest when to use it, and why, similiar to the language options above.

![image](https://github.com/streetsidesoftware/cspell-dicts/assets/32241933/f18e1c47-475d-4729-80a9-168deca222e3)

**Comment by Jason3S** [comment]
@KieranRobson,

Good point. A Trie is a data structure that is used to efficiently store large dictionaries with common suffixes. Which is what most natural languages have.
